### DISCUSSION TITLE
在iOS设备上使用绳网app

### DISCUSSION THREAD

**Discussion by Yizakl** [share121/inter-knot#4893] [教程]
![微信图片_20240915141331](https://github.com/user-attachments/assets/39fc99ae-ccac-48df-a617-6a0bda54ff96)

2024年11月9日
最后一次更新此篇文章：
-使用ChatGPT理清了顺序，更加容易阅读

2025年1月14日
祝大家新年快乐（迟来了喵）
在这里祝各位绳匠天天开心喵
-真的是最后一次更新了

---

### 重要提示：网盘中包含所有必要文件！点击链接下载：[网盘链接](https://www.123pan.com/s/Xpd6Vv-41Q6H?)  
**提取码：inot**  
*网盘内的ipa文件只更新到 2.16.0+27，这是最后一个稳定版（适用于移动端）。如果想使用最新版本，请前往 GitHub 仓库下载。*

## 准备工作：
你需要一台 **iPhone/iPad**（最好还准备一台 PC）。

---

### 1. 直接使用我提供的 IPA 文件
（文件可以在上面提供的网盘中找到）  
**注意**：开学了，可能跟不上每次的更新，更新时可以来官方 Discord 服务器找我（@yizakl）问询。

### 2. 手动构建 IPA 文件

1. 打开 **Inter-Knot** 项目的 GitHub 页面：[Inter-Knot 项目链接](https://github.com/share121/inter-knot/releases/tag/v2.16.0%2B27)

2. 找到最新版本，点击下载。

   ![image](https://github.com/user-attachments/assets/4ad875ed-11d1-4f1d-b916-af0b0df54f45)

3. 打开 **文件** 应用，找到下载文件夹（默认位置：iCloud 或者“我的 iPhone”中的下载文件夹）。

   ![image](https://github.com/user-attachments/assets/c8f0d929-7c2c-4593-9725-ead470e717ee)

4. 解压刚刚下载的文件，得到一个 `.app` 文件。

   ![image](https://github.com/user-attachments/assets/d84d5f59-2a0d-48b5-b782-655d51d4d2e5)

5. 创建一个名为 **Payload** 的文件夹，并将解压后的 `.app` 文件放进去。

6. 长按 **Payload** 文件夹，点击 **压缩**。

   ![image](https://github.com/user-attachments/assets/f533bf7b-ecf6-4dfc-ac26-ca95d80e0cc4)

7. 将文件后缀更改为 **.ipa**。

   ![image](https://github.com/user-attachments/assets/eeb256b7-24e3-41fa-a20e-81a5710a6f7e)

恭喜！你现在已经成功获取了绳网的 IPA 文件！

---

### 1. 查看设备是否支持 TrollStore

支持的设备列表：  
**iPhone：**  
- Arm64（a8）设备：仅限 iPad  
- Arm64（a9-a11）设备：iPhone 6s 到 iPhone X（支持 iOS 14.0beta2 到 17.0beta4）  
- Arm64e（A12-A17/M1/M2）设备：iPhone Xs 到 iPhone 15（支持 iOS 14.0beta2 到 16.7RC，17.0beta1-beat4）  

**iPad：**  
可以前往 [Apple 官网](https://www.apple.com.cn/ipad/compare/) 查看支持的设备型号。  
*注意：如果设备显示为 "Not Applicable" 或 "Unsupported"，请使用自签名方法。*

### 2. 安装 TrollStore 安装器
（可以通过签名安装、越狱安装或通过 DFU 模式安装，但不推荐后两种方法。）

如果你的设备支持 TrollStore，可以跳过安装 TrollInstallX 步骤，直接通过 **TrollHelperOTA** 进行安装。  
- 下载链接：[TrollHelperOTA 下载](itms-services:///?action=download-manifest&url=https://jailbreaks.app/cdn/plists/TrollHelper.plist)

---

## 三、通过自签名安装 IPA 文件

如果你不支持 TrollStore，可以通过 **自签名** 方法进行安装。

### 1. 安装 iTunes 或其他签名工具（如爱思助手）

1. 使用数据线将你的 iOS 设备连接到电脑。
2. 打开爱思助手并进入“工具箱”页面。

   ![image](https://github.com/user-attachments/assets/8d404e22-0c66-4faa-a439-b05609ad29b9)

3. 选择 **IPA 签名**，并使用 **Apple ID** 进行签名。

   ![image](https://github.com/user-attachments/assets/8e5b3a6c-a95b-472a-aa61-b8ad02416040)

4. 输入你的 Apple ID 和密码，如果开启了双重认证，请输入验证码。

5. 将 `TrollInstallX.ipa` 文件拖入签名工具，进行签名操作。  
   - 下载地址：[TrollInstallX.ipa 下载](https://github.com/alfiecg24/TrollInstallerX/releases/download/1.0.3/TrollInstallerX.ipa)

6. 签名完成后，找到生成的 IPA 文件并将其安装到你的设备上。

---

## 四、安装 IPA

1. 在 **文件** 应用中找到已经签名的 **绳网 IPA** 文件，长按并选择 **共享**。

   ![image](https://github.com/user-attachments/assets/89918240-65ef-4196-97dd-7cba8be356e2)

2. 选择 **TrollStore** 进行安装。

   ![image](https://github.com/user-attachments/assets/4d21f050-2385-4d96-a68b-e64169b558b9)

---

## 常见问题

**Q：TrollStore 安装失败？**  
**A**：重启设备后，再次打开 **TrollInstallX**，重复安装步骤。

**Q：需要开启开发者模式吗？**  
**A**：只有 iOS 16.0 及以上版本需要开启开发者模式。设置路径为：**设置 → 隐私与安全性 → 开发者模式**。

---

还有其他问题？可以在下面留言，看到后我会回复你的

**“窥视空洞之人，祝你得偿所愿。”**

---

**Comment by Yizakl** [comment]
我不是专业的，用词可能不准确...<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Yizakl** [comment]
没人看吗qwq

**Comment by share121** [comment]
我看了👀

**Reply by Yizakl** [reply]
666

**Comment by Signedinn** [comment]
感谢
![IMG_4298](https://github.com/user-attachments/assets/289ba1aa-597a-4253-908f-fcd4299bf1ef)

**Comment by CarlBobi** [comment]
我看了，非常感谢！已经搞定

**Reply by 323157364** [reply]
大佬强强强！！

**Comment by share121** [comment]
请问这个压缩，是压缩为 zip 文件吗？还是 7z 或 rar

**Reply by Yizakl** [reply]
抱歉，刚刚有事情没有看到，是压缩为.zip格式的

**Reply by Yizakl** [reply]
> 抱歉，刚刚有事情没有看到，是压缩为.zip格式的

实际上应该都可以，因为在安装ipa时，软件会自动帮你解压安装.app文件，但是.app文件TrollStore不会识别，只支持ipa
但是如果是套壳TrollStore（如轻松签+），在文件管理中会识别.app文件
![IMG_0089](https://github.com/user-attachments/assets/ce360dcc-cc7d-4684-a9bf-02ea0c62c38b)

**Reply by share121** [reply]
谢谢你，已经实现了 ipa 打包

**Comment by zzz20100624** [comment]
我丢强强强

**Comment by konghuan7759** [comment]
这是苹果手机的一个什么东西让这么做才能在苹果手机上用？

**Reply by Yizakl** [reply]
苹果主打封闭（能随意安装ipa会影响苹果赚米，因为苹果要吃苹果税，但是因为App Store的严格审核，也比其他应用商店干净），但也代表更多人会去钻漏洞，TrollStore是由@opa334 开发的一个开源的能随意安装ipa的软件：https://github.com/opa334/TrollStore
如果你愿意，可以给这位大大的项目点个star哦喵

**Comment by Yizakl** [comment]
qwq

**Comment by wushang114** [comment]
三楼惊现董事长

**Reply by Yizakl** [reply]
121照着我教程打包iOS版本的（自豪

**Comment by capacitor1** [comment]
会安装IOS软件，
# 你是大佬！
不接受反驳。

**Reply by Yizakl** [reply]
# 会用Markdown,我不会，你是大佬

**Comment by cicada3301digger** [comment]
佬，ios端怎么登录

**Reply by Yizakl** [reply]
🤔你是用新版还是旧版

**Reply by cicada3301digger** [reply]
今天才安装，应该是最新版？

**Reply by Yizakl** [reply]
我的意思是，你用的是我网盘的ipa还是仓库的ipa

**Reply by cicada3301digger** [reply]
github上面的

**Reply by Yizakl** [reply]
把手机横过来，点我的，选择用户接口
（左滑三次也可以）

**Reply by cicada3301digger** [reply]
woc，未曾设想的道路，谢谢佬

**Reply by Yizakl** [reply]
🤔
# 我是fw

**Comment by tanchideshaohuaya** [comment]
也可以iPhone/iPad连接电脑后打开爱思助手用ID自签，签完直接用爱思装就行

**Reply by Yizakl** [reply]
原本是有这句话的，被ChatGPT优化了，不想改了
